src/OrchardCore.Modules/OrchardCore.Contents/Views/Items/DeleteContentTask.Fields.Edit.cshtml

```diff
@@ -0,0 +1,8 @@
+@using OrchardCore.Contents.Workflows.ViewModels
+@model DeleteContentTaskViewModel
+
+<fieldset class="form-group" asp-validation-class-for="Expression">
+    <label asp-for="Expression">@T["Expression"] <span asp-validation-for="Expression"></span></label>
+    <input type="expression" asp-for="Expression" syntax="Script" autofocus />
```

Comment:
Explain `syntax=Script` ?

Reply:
The `syntax` attribute is used to control the label displayed here:

![image](https://user-images.githubusercontent.com/938393/35067895-c099de7a-fbd5-11e7-8c99-96d406d12ad2.png)

There's no actual logic behind it - anything you put in there appears on that label.

Reply:
Ok. Not a huge fan as it doesn't work for textarea. Would just use the description, which can also contain a link to the description of some samples when we have them in the doc.

Reply:
If that's the only concern, then it's [easy to make it work for textarea](https://stackoverflow.com/questions/24557828/bootstrap-input-group-textarea-with-button) as well. Description works, but that's kinda boring ;)